Neos.Diff/Classes/SequenceMatcher.php

```diff
@@ -597,7 +597,7 @@ public function getGroupedOpcodes($context = 3)
             ];
         }
 
-        if (!empty($group) && !(count($group) == 1 && $group[0][0] == 'equal')) {
+        if (!(count($group) == 1 && $group[0][0] == 'equal')) {
```

Comment:
this is a bit tricky but phpstan was really stubborn that this check is obsolete and it 100% is.

$group is never empty because it is filled in the foreach above where the variable $opCodes always have at least one entry.